patches/0016-implement-ms_tls_config_schannel-experiment.patch

```diff
@@ -238,6 +242,23 @@ index 00000000000000..a77186c66d3095
 +	return suites, versions, nil
 +}
 +
++// removeCipherSuiteCurvePrefix removes the curve prefix from the cipher suite name.
++// Windows Schannel supports cipher suites with curve prefixes, in the form of
++// _P256, _P384, or _P521, which are not present in the Go cipher suite names.
++// This is not common, as since Windows 2016 the curve prefix is discarded by Schannel,
++// but it can still be present in case Windows was upgraded from an earlier version.
++func removeCipherSuiteCurvePrefix(name string) string {
++	const prefixLength = 5 // Length of "_P256", "_P384", or "_P521"
++	if len(name) < prefixLength {
++		return name // No curve prefix to remove
++	}
++	switch name[len(name)-prefixLength:] {
++	case "_P256", "_P384", "_P521":
++		name = name[:len(name)-prefixLength] // Remove the curve prefix
```

Comment:
That's not a prefix, it's a suffix. ("Affix" being the general term.)

Reply:
d'oh, this is one of my achilles heels, I always swap them 😅